### PR TITLE
Feature/remove timer tags

### PR DESCRIPTION
**Pull request by siwane**
The purpose of this PR is to reduce the amount of data stored in QtiTimeLine by deleting couple of (unnecessary) tags

### CODE REVIEW

File: models/classes/runner/session/TestSession.php [hunk 1]
```diff
@@ -150,28 +150,14 @@ public function setTimerTarget($timerTarget)
...
-        $sections = $routeItem->getAssessmentSections();
-        $sections->rewind();
-        $sectionId = key(current($sections));
         $itemRef = $routeItem->getAssessmentItemRef();
         $itemId = $itemRef->getIdentifier();
         $occurrence = $routeItem->getOccurence();
 
-        $tags = [
-            $itemId,
+        return [
```
Comment (krampstudio) [inline models/classes/runner/session/TestSession.php]: Dead code after

File: models/classes/runner/session/TestSession.php [hunk 2]
```diff
@@ -150,28 +150,14 @@ public function setTimerTarget($timerTarget)
...
             $itemId . '#' . $occurrence,
-            $sectionId,
-            $testPart->getIdentifier(),
-            $test->getIdentifier(),
-            $itemRef->getHref(),
         ];
 
-        if ($this->isRunning() === true) {
-            $tags[] = $this->getItemAttemptTag($routeItem);
```
Comment (krampstudio) [inline models/classes/runner/session/TestSession.php]: I'd keep this one too, otherwise you won't be able to differentiate different attempts o

Comment (siwane) [reply models/classes/runner/session/TestSession.php]: I agree and I restore it, however this tag is the most heavy.

**Comment by jsconan** [comment]
Ok, the timers are working well now!

**Comment by jsconan** [comment]
It is also possible to reduce the size of the tags by removing the item id from the occurence and the attempt identifiers. The test id could also be removed. However this will need some small changes in the TimeLine usages.

Here is a sample of what the tags are currently looking: `[item-1, item-1#0, item-1#0-1, assessmentSection-4, testPart-4, Test-1]`.

Once the redundant ID have been removed, this sample could be: `[item-1, #0, #0-1, assessmentSection-4, testPart-4]`

**Comment by jsconan** [comment]
I created a TT story for the tags system improvement: https://oat-sa.atlassian.net/browse/TT-221

**Pull request merged**